content/html/concepts/tags/terms/center/center.md

```diff
@@ -0,0 +1,37 @@
+---
+Title: '<center>'
+Description: 'Depreciated block-level element that displays its contents centered horizontally in the containing element. CSS should be used instead.'
+Subjects:
+  - 'Web Development'
+  - 'Web Design'
+Tags:
+  - 'Tags'
+  - 'Style'
+CatalogContent:
+  - 'learn-html'
+  - 'paths/front-end-engineer-career-path'
+---
+
+The `<center>` tag was used historically to center elements, usually text, within their containing element. It has been removed from the HTML specification, and while some browsers might support it for backwards compatibility, it should no longer be used.
```

Comment:
```suggestion
The `<center>` tag was used historically to center content, usually text, within their containing elements. It has since been removed and deprecated from the HTML specification, and while some browsers might support it for backwards compatibility, it should no longer be used.
```